cdp_scrapers/instances/seattle.py

```diff
@@ -0,0 +1,359 @@
+                    name = attachment[LEGISTAR_FILE_NAME],
+                    uri = attachment[LEGISTAR_FILE_URI],
+                )
+            )
+
+        return files
+
+
+    @staticmethod
+    def get_matter(legistar_ev: Dict) -> Matter:
+        '''
+        cdp Matter from parts of legistar api EventItem
+        '''
+        return Matter(
+            external_source_id = legistar_ev[LEGISTAR_MATTER_EXT_ID],
+            name = legistar_ev[LEGISTAR_MATTER_NAME],
+            matter_type = legistar_ev[LEGISTAR_MATTER_TYPE],
+            title = legistar_ev[LEGISTAR_MATTER_TITLE],
+            result_status = legistar_ev[LEGISTAR_MATTER_STATUS],
+        )
+
+
+    @staticmethod
+    def get_event_minutes(legistar_ev_items: List[Dict]) -> List[EventMinutesItem]:
+        '''
+        return legistar 'EventItems' as EventMinutesItems
+        '''
+        # TODO: create more objects like Vote, SupportingFile, Person with available info
+
+        minutes = []
+
+        # EventMinutesItem object per member in EventItems
+        for item in legistar_ev_items:
+            minutes.append(
+                EventMinutesItem(
+                    decision = item[LEGISTAR_EV_MINUTE_DECISION],
+                    # other better choice for name?
+                    minutes_item = MinutesItem(name = item[LEGISTAR_MINUTE_NAME]),
+                    supporting_files = LegistarScraper.get_event_support_files(item[LEGISTAR_EV_ATTACHMENTS]),
+                    votes = LegistarScraper.get_votes(item[LEGISTAR_EV_VOTES]),
+                    matter = LegistarScraper.get_matter(item),
+                )
+            )
+
+        return minutes
+
+
+    @staticmethod
+    def legistar_ev_date_time(ev_date: str, ev_time: str) -> datetime.datetime:
+        '''
+        helper func combine legistar ev date and time into datetime
+        '''
+        # 2021-07-09T00:00:00
+        d = datetime.datetime.strptime(ev_date, '%Y-%m-%dT%H:%M:%S')
+        # 9:30 AM
+        t = datetime.datetime.strptime(ev_time, '%I:%M %p')
+        return datetime.datetime(
+            year = d.year, month = d.month, day = d.day,
+            hour = t.hour, minute = t.minute, second = t.second
+        )
+
+
+    def get_events(self,
+        # for the past 2 days
+        begin: datetime.time = datetime.datetime.utcnow() - datetime.timedelta(days = 2),
+        end:   datetime.time = datetime.datetime.utcnow()
+    ) -> List[EventIngestionModel]:
+        '''
+        main getter to retrieve legistar data as cdp ingestion model items
+        '''
+        evs = []
+
+        for legistar_ev in get_legistar_events_for_timespan(
+            self.client_name,
+            begin = begin,
+            end = end,
+            # NOTE: use the range below to retrieve events with 1 containing video
+            #begin = datetime.datetime(year = 2021, month = 7, day = 9),
+            #end = datetime.datetime(year = 2021, month = 7, day = 10),
```

Comment:
Eh. I would remove this and add as documentation.

Reply:
It would be rare for someone to edit the code but if it is docstring the user can provide and run.